train.py

```diff
@@ -43,7 +43,7 @@ def get_arguments():
                         help='Directory in which to store the logging '
                         'information for TensorBoard. '
                         'If the model already exists, it will restore '
-                        'the state and will continue training. '
+                        'the state and will continue training.'
```

Comment:
Yes, my bad... I checked the other cases where this happens, but didn't see this one. Sorry about that.